src/components/NavBar/NavBar.jsx

```diff
@@ -1,34 +1,50 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 
-import './NavBar.css';
+import styles from './NavBar.module.css';
 import FYALogo from '../../assets/fya-logo.png';
+import PlaceHolderPFP from '../../assets/placeholder_pfp.svg';
 
-function NavBar() {
+const AdminLinks = () => (
+  <>
+    <Link to="/add-box-form">Add Box</Link>
+    <Link to="/upload-csv">Upload CSV</Link>
+    <Link to="/export-csv">Export CSV</Link>
+  </>
+);
+
+const UserLinks = () => (
+  <>
+    <Link to="/">Relocate a Box</Link>
+    <Link to="/">Pick Up a Box</Link>
+  </>
+);
+
+const NavBar = ({ isAdmin }) => {
   return (
-    <div className="nav-bar">
-      <div className="fya-logo">
+    <div className={styles['nav-bar']}>
+      <div className={styles['fya-logo']}>
         <img src={FYALogo} alt="Find Your Anchor Logo" />
       </div>
```

Comment:
Could we add a Link tag for this image to link to the home page ('/') when clicked?